uwblueprint/community-fridge-kw
# feat: Added admin donation filter buttons

## Brief description. What is this change? 

### [Create Admin Scheduled Donations Filter Views and Buttons](https://www.notion.so/uwblueprintexecs/Create-Admin-Scheduled-Donations-Filter-Views-and-Buttons-40949fd284da4adfb74b0e31c0a70634)

## Implementation description. How did you make this change?
* Added admin donation views export and filter buttons
* Added frontend filter functionality for admin donation view (gets all donations on first render and filter on FE)
* Fixed mobile + desktop spacing for donation and check in views


## Steps to test
1. Create two donations 
2. Create a volunteer account
3. Use PG Admin to assign a volunteer to one of the two donations by editing directly when viewing the table 
4. Log in as an admin
5. Go to Fridge Management > Scheduled Donations
6. All scheduled donations should show as default filter is All
7. Select filter by Filled
8. Only the scheduled donation with the assigned volunteer should show
9. Select filter by Unassigned
10. Only the scheduled donation with no assigned volunteers should show
11. Make sure mobile + desktop matches designs and check for the Check-In view as well


## Checklist
